Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -4749,107 +4067,6 @@ - (nullable NSIndexPath *)firstIndexPathAtViewHorizonTimestamp
     }
 }
 
-// We stop observing database modifications when the app or this view is not visible
-// (see: shouldObserveDBModifications).  When we resume observing db modifications,
-// we want to extend the "range" of this view to include any items added to this
-// thread while we were not observing.
-- (void)extendRangeToIncludeUnobservedItems
```

Comment:
We no longer need to do this, since the VM continues to update when the view is hidden.